### ISSUE TITLE
Date slider a11y issues

### ISSUE THREAD

**Issue by jvine** [sul-dlss/exhibits#1812]
The date range histogram and slider are not accessible. A keyboard/SR user can access and complete the from/to range and apply it. However, a non-sighted or keyboard-only sighted user cannot learn the distribution of results across the current date range. SR can perceive that there's an unlabelled image, and can read the marker dates (1800, 1900, 2000...) without context, but cannot perceive the ranges and results represented in the histogram. Keyboard cannot access the histogram to activate the tooltips, nor the slider to adjust the dates in that way.

(Bonus: when a mouse user has SR playing, clicking on the histogram reads apparently random numbers completely unrelated to the visible tooltips.)

An accessible approach might look something like this:
- structure the data (ranges + results) in a table
- present that table visually as a histogram, but allow it to be read by the SR (and even a sighted user, if they want) as tabular data
- use a keyboard-accessible slider
but this is likely out of scope for now / in a component we don't control.

Currently, this section is a bit mystifying when it's encountered by keyboard + SR. 

Mitigation proposal for now, until we can implement a better solution:

- [x] 1. Hide the histogram and slider from SR by applying `aria-hidden=”true”`
- [x] 2. Expose a currently hidden text element to SR by changing `style="display:none"` to `class="sr-only"`
    ```
    <p class="range subsection slider_js">
           <div style="display:none">
            Current results range from <span class="min">700</span> to <span class="max">1799</span>
          </div>
   ```
- [x] 3. Delete an unattached label that is adding confusion:
`<label class="range_limit_label sr-only">Date range:</label>`

IMO, this will eliminate noise from the elements we can't change _for now_, and provide a little more context to the SR.
